rln/src/protocol.rs

```diff
@@ -114,8 +127,9 @@ pub fn serialize_witness(rln_witness: &RLNWitnessInput) -> Result<Vec<u8>> {
     serialized.extend_from_slice(&fr_to_bytes_le(&rln_witness.identity_secret));
     serialized.extend_from_slice(&fr_to_bytes_le(&rln_witness.user_message_limit));
     serialized.extend_from_slice(&fr_to_bytes_le(&rln_witness.message_id));
-    serialized.extend_from_slice(&vec_fr_to_bytes_le(&rln_witness.path_elements)?);
-    serialized.extend_from_slice(&vec_u8_to_bytes_le(&rln_witness.identity_path_index)?);
+    // Note: safe to unwrap (error is ())
+    serialized.extend_from_slice(&vec_fr_to_bytes_le(&rln_witness.path_elements).unwrap());
```

Comment:
as I see this function doesn't have error at all, maybe we remove returning result?

Reply:
Fixed